test/end-to-end/pages/executors/number.js

```diff
@@ -1,17 +1,23 @@
 'use strict';
 
 const config = require('config');
-const commonContent = require('app/resources/en/translation/common');
-const content = require('app/resources/en/translation/executors/number');
+const commonContentEn = require('app/resources/en/translation/common');
+const commonContentCy = require('app/resources/cy/translation/common');
+const pagePath = require('app/steps/ui/executors/number');
```

Comment:
This is undoing performance work - including these files takes time - do we need to?